modules/relevantDigitalBidAdapter.js

```diff
@@ -0,0 +1,194 @@
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+import {ortbConverter} from '../libraries/ortbConverter/converter.js'
+import {BANNER, NATIVE, VIDEO} from '../src/mediaTypes.js';
+import {config} from '../src/config.js';
+import {pbsExtensions} from '../libraries/pbsExtensions/pbsExtensions.js'
+import {deepSetValue, isEmpty, deepClone, shuffle, triggerPixel, deepAccess} from '../src/utils.js';
+
+const BIDDER_CODE = 'relevantdigital';
+
+const converter = ortbConverter({
+  context: {
+    netRevenue: true,
+    ttl: 300
+  },
+  processors: pbsExtensions,
+  imp(buildImp, bidRequest, context) {
+    // Set stored request id from placementId
+    const imp = buildImp(bidRequest, context);
+    const { placementId } = bidRequest.params;
+    deepSetValue(imp, 'ext.prebid.storedrequest.id', placementId);
+    delete imp.ext.prebid.bidder;
+    return imp;
+  },
+});
+
+/** Global settings per bidder-code for this adapter (which might be > 1 if using aliasing) */
+let configByBidder = {};
+
+/** Used by the tests */
+export const resetBidderConfigs = () => {
+  configByBidder = {};
+};
+
+/** Settings ber bidder-code. checkParams === true means that it can optionally be set in bid-params   */
+const FIELDS = [
+  { name: 'pbsHost', checkParams: true, required: true },
```

Comment:
In the foreseeable future we'll need to continue use different domains. The "problem" is that our Prebid Server instances are already up and running - using different domain names.
The use-case for this adapter is that some existing network-customers should be able to let publishers use "just another" Prebid adapter instead of forcing them to run our normal js-wrapper on their sites.